### PR TITLE
Add the merging steps to the README

### PR DESCRIPTION
**Pull request by melaniebrd**
In order to help the developers have the same protocol to merge
new changes, we wrote the steps into the README.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,5 @@
 *.ear
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
```
Comment (ghost) [inline .gitignore:12]: Why did you delete this line ?

Comment (melaniebrd) [reply .gitignore:12]: Oops I'll amend this change :p
